services/queue/src/api.js

```diff
@@ -1745,13 +1731,10 @@ builder.declare({
 }, async function(req, res) {
   const {provisionerId, workerType} = req.params;
 
+  const expires = new Date();
   const [wType, provisioner] = await Promise.all([
-    this.WorkerType.load({
-      provisionerId,
-      workerType,
-      expires: Entity.op.greaterThan(new Date()),
-    }, true),
-    this.Provisioner.load({provisionerId}, true),
+    WorkerType.get(this.db, provisionerId, workerType, expires),
+    Provisioner.get(this.db, provisionerId, expires),
```

Comment:
There must have been a reason for adding `expires`. I will stop commenting on this :)